.gitignore

```diff
@@ -277,3 +277,9 @@ gallery/how_to/work_with_microtvm/micro_tvmc.py
 
 # GDB history file
 .gdb_history
+
```

Comment:
would it makes sense to output these files into the build/ directory and add that directory as an include instead of putting these files into the source tree?